src/transformers/pipelines/__init__.py

```diff
@@ -415,7 +415,8 @@
 # any tokenizer/feature_extractor might be use for a given model so we cannot
 # use the statically defined TOKENIZER_MAPPING and FEATURE_EXTRACTOR_MAPPING to
 # see if the model defines such objects or not.
-MULTI_MODEL_CONFIGS = {"SpeechEncoderDecoderConfig", "VisionEncoderDecoderConfig", "VisionTextDualEncoderConfig"}
+MULTI_MODEL_SPEECH_CONFIGS = {"SpeechEncoderDecoderConfig"}
```

Comment:
Do we mean multi-modal here?

Reply:
I'm not sure, I think it refers to multiple models (audio model and text model for instance).